src/components/BorrowerProfile/LendCta.vue

```diff
@@ -59,6 +59,7 @@
 							@change="$kvTrackEvent(
 								'Lending',
 								'Modify loan amount',
+								'priceOption from below?'
```

Comment:
We normally collect the amount in the 4th (value) + 5th (property) params as whole numbers.